teachertool/src/services/indexedDbService.ts

```diff
@@ -104,25 +78,23 @@ const getDb = (async () => {
     return db;
 })();
 
-export async function getLastActiveRubricAsync(): Promise<Rubric | undefined> {
+export async function getRubricFromIndexedDbAsync(name: string): Promise<Rubric | undefined> {
```

Comment:
I saw that you mentioned this in your PR comment. I don't think this needs to have the `fromIndexedDb` in the name since you have to access the function from the indexedDbService. And the functions that you have for localstorage only involve the name, so I don't think there's really room for confusion there.

Reply:
For this one, I think it's less necessary since there isn't a helper for it, but I think it is necessary for the others (we don't want someone calling saveRubric here instead of the helper for it), and then it's just a matter of consistency.

Reply:
Gotcha, that makes sense.